### PR TITLE
CYB-550 longer producer schedule increase delay

### PR DESCRIPTION
**Pull request by adrianopol**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -65,10 +65,11 @@ endif()
 #TODO: Contracts' tests depend on different libraries for different build types:
 # libfc.a (for release) or libfc_debug.a (for debug) (the same for secp256k1 library).
 # (see EosioTester.cmake)
+set(TEST_BUILD_TYPE RelWithDebInfo)
+#~set(TEST_BUILD_TYPE ${CMAKE_BUILD_TYPE})
```
Comment (justefg) [inline CMakeLists.txt:69]: comment ?

Comment (adrianopol) [reply CMakeLists.txt:69]: Commented version should be left after adding both variants -- debug & releare -- for libfc and libsecp256k1 libraries to the daocasino/daobet-with-cdt docker image.

File: contracts/eosio.system/src/voting.cpp [hunk 2]
```diff
@@ -74,40 +74,54 @@ namespace eosiosystem {
...
-          target_schedule_size = target_schedule_size + _gstate.schedule_size_step;
-        } else if (target_amount < target_schedule_size) {
-          target_schedule_size = target_schedule_size - _gstate.schedule_size_step;
-        }
-        _gstate.last_target_schedule_size_update = block_time;
-        _gstate.target_producer_schedule_size = target_schedule_size;
+      const int32_t new_target_schedule_size = get_target_schedule_size(activated_share);
+
+      // try to update schedule size every schedule_decrease_delay_sec seconds
+      if (block_time.slot - _gstate4.last_schedule_size_decrease.slot >= 2 * _gstate4.schedule_decrease_delay_sec) {
```
Comment (justefg) [inline contracts/eosio.system/src/voting.cpp:97]: what about `block_time.slot - _gstate4.last_schedule_size_increase >= ... ` case ?

Comment (adrianopol) [reply contracts/eosio.system/src/voting.cpp:97]: @justefg, split into two independent cases.

**Pull request merged**